src/Sonarr.Api.V3/Episodes/RenameEpisodeController.cs

```diff
@@ -26,5 +27,17 @@ public List<RenameEpisodeResource> GetEpisodes(int seriesId, int? seasonNumber)
 
             return _renameEpisodeFileService.GetRenamePreviews(seriesId).ToResource();
         }
+
+        [HttpGet("multiple")]
+        [Produces("application/json")]
+        public List<RenameEpisodeResource> GetEpisodes(List<int> seriesIds)
+        {
+            if (seriesIds is not { Count: not 0 })
+            {
+                throw new BadRequestException("seriesIds must be provided");
+            }
+
```

Comment:
Another check that none of the IDs are `0` would also be good, otherwise it leads to a 500 response.